### DISCUSSION TITLE
API Definition Using WSDL

### DISCUSSION THREAD

**Discussion by haozhen3513** [apiman/apiman#2583] [Help]
Hi all,

I would like to have an API definition for my soup request using WSDL.

I already uploaded the WSDL to the API definition.

The WSDL I am trying to use was copy from here.

https://www.tutorialspoint.com/wsdl/wsdl_example.htm

![image](https://github.com/apiman/apiman/assets/88840693/35109f14-b828-42c7-b9d8-c5e9ac272ac9)

<img width="813" alt="image" src="https://github.com/apiman/apiman/assets/88840693/4fba7742-f9c1-4dcb-88fc-ccea9dcab0a8">

It doesn't work and display This API does not have an API definition file. Contact the API provider and ask them to supply a valid API definition!

Your answer was highly appreciated!!!

**Comment by volkflo** [comment]
Currently there is no UI for WSDL files, like we have the Swagger UI for Open-API definitions.
AFAIK there is also no project that provides a UI where you can make test requests like you can in swagger ui.

You can only get the WSDL by the provided URL in the orange box and view in the browser as file or download and use a editor of you choice.
